src/decorator.ts

```diff
@@ -58,26 +63,34 @@ export default function ReactN<
       }
     }
 
-    public UNSAFE_componentWillUpdate(...args: [ P, S, any ]): void {
-      const [ rVerMaj, rVerMin ] = version.split('.').map((v): number => parseInt(v));
-      if (rVerMaj > 16 || (rVerMaj === 16 && rVerMin >= 3)) {
+    public componentWillUpdate(...args: [ P, S, any ]): void {
```

Comment:
I think this will console log ("Use UNSAFE instead"), since you are defining this method. This method should only _exist_ if `isUsingOldReact`.